ansible_image_validation/validation-playbooks/ade_packages_validation.yaml

```diff
@@ -0,0 +1,29 @@
+###
+# This Playbook checks if the packages required by Azure Disk Encryption is installed 
+###
+
+---
+
+- name: Check if the packages required by Azure Disk Encryption is installed
+  command: rpm -q "{{ item }}"
```

Comment:
Why are we using ```command``` here ? There is ansible way of doing this.
Instead just, error this way if the package is not installed.
``` yaml
- name: "NGINX test result"
  debug: 
    msg: "NGINX NOT found"
  when: "'nginx' not in ansible_facts.packages"
```

Just enclose this in a loop similar to how command is executed. Killing two birds with one stone :+1:

Reply:
I tried earlier but wasn't able to use lineinfile, when and loop clause together